src/main/telemetry/srxl.c

```diff
@@ -115,6 +114,9 @@ void srxlFrameQos(sbuf_t *dst)
     sbufWriteU16BigEndian(dst, 0xFFFF); // F
     sbufWriteU16BigEndian(dst, 0xFFFF); // H
     sbufWriteU16BigEndian(dst, 0xFFFF); // rxVoltage
+    return true;
+
+    UNUSED(currentTimeUs);
```

Comment:
Convention is to put `UNUSED` at the top of the function.

Reply:
OK, fixed.